public/vendor/librejs.html

```diff
@@ -16,14 +16,14 @@
           <td><a href="https://github.com/codedance/jquery.AreYouSure/archive/1.9.0.tar.gz">jquery.areyousure-1.9.0.tar.gz</a></td>
         </tr>
         <tr>
-          <td><a href="./plugins/jquery/jquery.min.js">jquery.min.js</a></td>
+          <td><a href="../js/jquery.js">jQuery</a></td>
           <td><a href="https://jquery.org/license/">MIT</a></td>
           <td><a href="https://code.jquery.com/jquery-3.4.1.min.js">jquery-3.4.1.min.js</a></td>
         </tr>
         <tr>
-          <td><a href="./plugins/jquery-migrate/jquery-migrate.min.js">jquery-migrate.min.js</a></td>
+          <td><a href="../js/jquery.js">jQuery Migrate</a></td>
```

Comment:
Is this href OK? I don't know what the requirements are for `librejs.html`.

Reply:
It should be. The file is served at `public/librejs.html`: https://try.gitea.io/vendor/librejs.html

I'd like to eventually get red of this file and use something auto-generated.